src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -150,6 +150,8 @@
 				<!-- See https://github.com/xamarin/xamarin-macios/issues/14812 for more details. -->
 				<MtouchExtraArgs>$(MtouchExtraArgs) --marshal-objectivec-exceptions:disable</MtouchExtraArgs>
 				<CodesignEntitlements>iOS\Entitlements.plist</CodesignEntitlements>
+				<!-- Workaround for clang++ error on iOS simulator on ARM base macOS system -->
```

Comment:
`<!-- Workaround for clang++ error on iOS simulator on ARM base macOS system. See "LINK of your source/issue" for  more details. -->`